client/post-editor/media-modal/gallery/edit.jsx

```diff
@@ -40,24 +41,44 @@ export default React.createClass( {
 	},
 
 	render() {
-		const { site, settings } = this.props;
+		const { onUpdateSetting, site, settings, translate } = this.props;
 
 		if ( ! site || ! settings.items ) {
 			return null;
 		}
 
+		const orders = {
+			[ translate( 'Reverse order' ) ]: reverse( [ ...settings.items ] ),
+			[ translate( 'Order alphabetically' ) ]: sortBy( settings.items, 'title' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Alphabetical Order' )` *ES Score: 18*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/media-modal-sort-order-dropdown)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).